koap/build.gradle.kts

```diff
@@ -11,7 +11,7 @@ apply(from = rootProject.file("gradle/jacoco.gradle.kts"))
 
 kotlin {
     jvm()
-    js().browser()
```

Comment:
Oh, was this important?

Reply:
Yup. In future versions either `node()` or `browser()` will be required. Right now it's just a warning if you omit it.

Reply:
Putting it back. ^_^